drafting 2.0.0 release notes

elrido opened the pull request:
closes #43 - please review and comment or commit your changes to the branch, thanks!

The date is still provisional and I'll update and merge this once we are ready for that release.

elrido committed:
drafting 2.0.0 release notes

Ribas160 commented on content/2025-07-31-v2.0.0-release.md:
```diff
@@ -0,0 +1,94 @@
+
+**This release changes configuration defaults including switching the template and removing legacy features.**
+
+The most notable change is the switch of the default template to `bootstrap5`. We also switch to the Jdenticons library by default for the comment creator icons, as it doesn't require the GD library.
```
```suggestion
The most notable change is the switch of the default template to `bootstrap5`. We also switched to the Jdenticons library by default for the comment creator icons, as it doesn't require the GD library.
```

elrido committed:
wording

Co-authored-by: Mikhail Romanov <42250412+Ribas160@users.noreply.github.com>

Ribas160 commented on content/2025-07-31-v2.0.0-release.md:
```diff
@@ -0,0 +1,94 @@
+The most notable change is the switch of the default template to `bootstrap5`. We also switched to the Jdenticons library by default for the comment creator icons, as it doesn't require the GD library.
+
+The `page` template and compatibility to pre-v1.3 pastes, including ZeroBin ones, got removed. The `v2` paste format which is still currently used, got introduced in release 1.3 in July 2019. This allows dropping further unused database columns, as those were artifacts of the `v1` pastes and not used anymore with the `v2` ones.
+
```
```suggestion

The minimum required PHP version was increased from 7.3 to 7.4.

```

Ribas160 replied:
The PHP version upgrade is mentioned in the changelog, but we might consider including it in the main description, as it could be a critical change for some users

https://github.com/PrivateBin/privatebin.info-pelican/blob/01ee198816450bb561ffc5fade3928c24db16e43/content/2025-07-31-v2.0.0-release.md?plain=1#L58

elrido committed:
update changelog with proposed, latest changes

When we skipped patch versions between release notes, we include all version changes since the last release note document. From 1.6.0 on, we only published these on major or minor releases and skip creating a note for every patch release.

elrido committed:
mention the new minimum PHP version

Co-authored-by: Mikhail Romanov <42250412+Ribas160@users.noreply.github.com>

elrido committed:
additional visible changes

elrido committed:
belongs more into upgrade section

elrido committed:
updated from README.md in main repo

elrido committed:
updated screenshots page wording, removed page template

Ribas160 commented:
How do you think, should we add notes that the UI terminology and website template have been updated?

elrido committed:
update date of release

elrido commented:
The UI terminology change is at least mentioned in the changelog and I assumed the project website template change would be explained implicitly due to us changing the default in the configuration and mentioning this as the most noticeable change.

We can still change this doc later on, but I'm now already in the middle of the release process and will conclude that work as is.
